webview/src/experiments/components/table/styles.module.scss

```diff
@@ -785,7 +788,7 @@ $bullet-size: calc(var(--design-unit) * 4px);
   text-align: left;
 }
 
-.thead {
```

Comment:
I can no longer decrease the column width to be shorter than the text width itself: 

![image](https://user-images.githubusercontent.com/43496356/213747844-f2706077-8aca-495e-8fb5-2b620b614145.png)

Reply:
Fixed!